src/Test.sol

```diff
@@ -226,58 +230,32 @@ abstract contract Test is DSTest, Script {
     }
 
     function assertEq(uint256[] memory a, uint256[] memory b) internal {
-        if (a.length != b.length) {
-            emit log("Error: a.length == b.length not satisfied");
+        if (keccak256(abi.encode(a)) != keccak256(abi.encode(b))) {
+            emit log("Error: a == b not satisfied [uint[]]");
+            emit log_named_array("  Expected", b);
+            emit log_named_array("    Actual", a);
```

Comment:
Forge doesn't support printing these, does it? @mds1 

They show up only in traces.

Reply:
Thanks for handling the tests!

Good point, it doesn't currently, but it shouldn't be too hard to support so we can create an issue upstream to track